packages/webchat/src/index.tsx

```diff
@@ -46,6 +46,7 @@ class ExposedWebChat extends React.Component<Props, State> {
   }
 }
 
+// TODO: what does this observer do?
 const Wrapper = observer(ExposedWebChat)
 
```

Comment:
@allardy Any comments on this? I don't use the wrapper at the moment and I haven't see much difference.

Reply:
When you have those kind of questions, you can hover on the line and it will display the related commit, giving a reason why this was added.
![image](https://user-images.githubusercontent.com/42552874/141335580-8b691435-a566-4660-9959-79a647f68b87.png)

Since you hardcoded english on the fetch preferences, you might not have seen it